mcp-lambda-server/tests/validation/test_validation_engine.py

```
@@ -0,0 +1,563 @@
+        assert "불변" in setter_suggestion.suggestion or "final" in setter_suggestion.suggestion
+
+    def test_validate_and_regenerate_pass_first_attempt(self):
+        """첫 시도에 통과하는 경우"""
+        engine = get_validation_engine()
+        code = """
+        public class Order {
+            private final Long id;
+            public Order(Long id) { this.id = id; }
+        }
+        """
+        result, final_code, attempts = engine.validate_and_regenerate(code, "DOMAIN")
```

Ruff (0.14.13)

Unpacked variable `final_code` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)